### PR TITLE
RBMK - D.I.Y Nuclear reactor! (new engine)

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request
Through extensive testing on bungalow, This in fact works, however the Cargo RBMK doesn't, it looks like a issue with linking with a multitool, and I haven't written down how to set up it yet.
image

Heres a image someone made to show how to set it up on the test map/Deltastation RBMK edition
image
although maxing out the top is safer

## Originial PR text
ports the rbmk from oasis, which in turn is a port from austation, which IN TURN is a port from NSV
https://github.com/Oasis-SS13/Oasis-SS13/pull/714
image

## Why It's Good For The Game
we are getting singulo, teg, and tesla back, so why a extra engine?

Also can be found by void techs, and maybe orderable from cargo?

okay so issues

- [ ] tgui is buggy (coolant line graph is broken, coolant input temp doesn't show, window starts real small, can resize)
- [x] reactor power goes to negative(?!)
- [x] outputs no power
- [x] PSI can get to a ridiculous amount
- [ ] computers need better sprites
- [ ] and probably 10 other bugs i never found out

...but it compiles with no ~~runtimes~~! not anymore 
todo

- [x] make a rbmk test map instead of torturing runtime station for testing
- [x] make a ruin to find the rbmk capsule
- [ ] ~~remove rbmk from cargo~~

this probably wasn't a good beginner project

## Changelog
:cl:
add: Nanotrasen would like to remind employees that although over 500 years old, the RBMK is a safe engine that has stood the test of time.
soundadd: rbmk touched sound code

/:cl:

### CODE REVIEW

File: ModularTegustation/tegu_rmbk/code/modules/power/reactor/rbmk.dm [hunk 1]
```diff
@@ -0,0 +1,974 @@
...
+			last_power_produced *= base_power_modifier //Finally, we turn it into actual usable numbers.
+			radioactivity_spice_multiplier += moderator_input.gases[/datum/gas/tritium][MOLES] / 5 //Chernobyl 2.
+			var/turf/T = get_turf(src)
+			if(power >= 20)
+				coolant_output.gases[/datum/gas/nitryl][MOLES] += total_fuel_moles/50 //Shove out nitryl into the air when it's fuelled. You need to filter this off, or you're gonna have a bad time.
+			var/obj/structure/cable/C = T.get_cable_node()
+			if(!C || !C.powernet)
+				return
+			else
+				C.powernet.newavail += last_power_produced
```
Comment (EgorDinamit) [inline ModularTegustation/tegu_rmbk/code/modules/power/reactor/rbmk.dm:274]:
```suggestion
			add_avail(last_power_produced)
```
Try this, it's a parent proc.

**Comment by EgorDinamit** [comment]
Oh no.

**Comment by BootlegBow** [comment]
watch me speedrun chernobyl station 13

**Comment by retlaw34** [comment]
i think this would be easier if i wasn't using notepad++

**Comment by retlaw34** [comment]
cargo rbmk doesn't work, alt delta doesn't show up on map select, re-drafting

**Comment by retlaw34** [comment]
Eh its a little broken, and not very proud of the code i did here. I will redo it along with some extra goodies when i get some more time.

**Comment by rye-rice** [comment]
i can no longer endorse this

**Pull request closed without merging**